src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in inversedRobot) {
+      return null;
+    } else {
+      inversedRobot[robot[key]] = key;
+    }
```

Comment:
```suggestion
    }
    
    inversedRobot[robot[key]] = key;
```
you don't need else, since there is return statement